source/core/coverage/ut_coverage_helper.pkb

```diff
@@ -59,17 +68,18 @@ create or replace package body ut_coverage_helper is
   end;
 
   procedure coverage_stop is
-    l_return_code binary_integer;
   begin
     if not g_develop_mode then
-      l_return_code := dbms_profiler.stop_profiler();
+      g_is_started := false;
+      dbms_profiler.stop_profiler();
     end if;
   end;
 
   procedure coverage_stop_develop is
-    l_return_code binary_integer;
   begin
-    l_return_code := dbms_profiler.stop_profiler();
+    g_develop_mode := false;
```

Comment:
We can't, as described in previous comment.
The start_develop/stop_develop override and block start/stop of coverage gathering.